Kysect.BotFramework/ApiProviders/IBotApiProvider.cs

```diff
@@ -14,5 +14,7 @@ public interface IBotApiProvider
         Result<string> SendText(string text, SenderInfo sender);
         public Result<string> SendMedia(IBotMediaFile mediaFile, string text, SenderInfo sender);
         public Result<string> SendMultipleMedia(List<IBotMediaFile> mediaFiles, string text, SenderInfo sender);
+
+        public Result<string> SendOnlineMedia(IBotOnlineFile file, string text, SenderInfo sender);
```

Comment:
```suggestion
        Result<string> SendMedia(IBotMediaFile mediaFile, string text, SenderInfo sender);
        Result<string> SendMultipleMedia(List<IBotMediaFile> mediaFiles, string text, SenderInfo sender);
        Result<string> SendOnlineMedia(IBotOnlineFile file, string text, SenderInfo sender);
```

Reply:
```suggestion
        Task<Result<string>> SendMediaAsync(IBotMediaFile mediaFile, string text, SenderInfo sender);
        Task<Result<string>> SendMultipleMediaAsync(List<IBotMediaFile> mediaFiles, string text, SenderInfo sender);
        Task<Result<string>> SendOnlineMediaAsync(IBotOnlineFile file, string text, SenderInfo sender);
```

Бтв не понимаю в чем прикол тащить Result везде подряд, ну он же нахрен тут не нужен и только мешается? Я уже задолбался смотреть на строчки ```if(result.IsFaulted) return;```

Reply:
@annchous Видала да как умею?

Reply:
> @annchous Видала да как умею?

ага